GenumKit/Stencil/Contexts.swift

```diff
@@ -151,7 +151,8 @@ extension StringsFileParser {
           "count": entry.types.count,
           "types": entry.types.map { $0.rawValue },
           "declarations": (0..<entry.types.count).map { "let p\($0)" },
-          "names": (0..<entry.types.count).map { "p\($0)" }
+          "names": (0..<entry.types.count).map { "p\($0)" },
+          "typednames": (0..<entry.types.count).map { "p\($0): \(entry.types[$0].rawValue)" }
```

Comment:
You can probably use `enumerate` instead here, like `entry.types.enumerate().map { p\($0): \($1.rawValue) }`